etl-pipeline/api/db.py

```diff
@@ -185,36 +185,39 @@ def fetchNewWorks(self, page=0, size=50):
         return (baseQuery.count(), baseQuery.offset(offset).limit(size).all())
 
     def fetchSingleCollection(self, uuid):
-        return (
-            self.session.query(Collection)
-                .options(
-                    joinedload(Collection.editions),
-                    joinedload(Collection.editions, Edition.links),
-                    joinedload(Collection.editions, Edition.items),
-                    joinedload(Collection.editions, Edition.items, Item.links),
-                    joinedload(Collection.editions, Edition.items, Item.rights),
-                )
-                .filter(Collection.uuid == uuid).one()
-        )
+        return self._query_collections().filter(Collection.uuid == uuid).one()
 
     def fetchCollections(self, sort=None, page=1, perPage=10):
         offset = (page - 1) * perPage
 
-        sort = sort.replace(':', ' ') if sort else 'title'
+        # The `sort` param is either `title` or `creator`, optionally with a sort direction
+        # of `asc` or `desc` appended with a colon.  However, we can't just pass a plain
+        # string to sqlalchemy, as the `title` field is ambiguous across different entities.
+        # So instead, turn the sort field into a proper sort-clause on the Collection table.
+        sort_field, *suffix = sort.split(":")
+        sort_clause = getattr(Collection, sort_field)
+        if suffix and suffix[0] == "desc":
+            sort_clause = sort_clause.desc()
```

Comment:
I didn't add an `else` for `asc` since that's the default sort direction in postgres